ERCS/erc-7549.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7549
```

Comment:
```suggestion
eip: 7588
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Updated. Thanks for your comments. :)